src/plugin.js

```diff
@@ -44,25 +44,20 @@ const errors = {
 /**
  * Initializes the plugin
  */
-const vr = function(options) {
-  const player = this;
+const initPlugin = function(player, options) {
 
   // don't initialize twice
-  if (this.usingPlugin('vr')) {
+  if (player.vr && player.vr.currentProjection) {
```

Comment:
I think that I may just have had the if check in the wrong place. It should probably go below all the error checking return statements. I also think that having it here would probably work now because the plugin waits for `player.ready`